inc/umock_c/umocktypes_wcharptr.h

```diff
@@ -26,6 +26,8 @@ extern "C" {
     int umocktypes_copy_const_wcharptr(const wchar_t** destination, const wchar_t** source);
     void umocktypes_free_const_wcharptr(const wchar_t** value);
 
+    void static mbstate_t_init(mbstate_t* ps);
```

Comment:
>void static mbstate_t_init(mbstate_t* ps); [](start = 4, length = 42)

This should be removed from the header as mbstate_t_init is static and internal to umocktypes_charptr.

Reply:
deleted from header